src/api/services/queries/ledger.ts

```diff
@@ -42,27 +42,29 @@ export function useCreateToken(neighborhood: Network | undefined) {
 
   return useMutation(
     async ({
-      address,
       amount,
+      destination,
       name,
-      symbol,
+      owner,
       precision = 9,
+      symbol,
     }: {
-      address: string;
       amount: string;
+      destination: string;
       name: string;
-      symbol: string;
+      owner: string;
       precision?: number;
+      symbol: string;
     }) =>
       await neighborhood?.tokens.create({
         summary: {
           name,
           symbol: symbol.toUpperCase(),
-          precision: 9,
+          precision,
         },
-        owner: address,
+        owner,
         distribution: {
-          [address]: BigInt(parseInt(amount) * 10 ** precision),
+          [destination]: BigInt(parseInt(amount) * 10 ** precision),
```

Comment:
...while the destination of the initial distribution will be set to the `destination` argument.